tests/TestCase/Command/I18nExtractCommandTest.php

```diff
@@ -58,6 +64,8 @@ public function tearDown(): void
         $fs = new Filesystem();
         $fs->deleteDir($this->path);
         $this->clearPlugins();
+
+        Configure::write('App.paths', $this->configPaths);
```

Comment:
You shouldn't need to do this backup/restore. The `TestCase::setup()` and `teardown` methods already do this.

Reply:
Makes sense, yes. I took it from here: https://github.com/cakephp/cakephp/blob/6aeb856ee523d326567d745a69db9ee7c9c7d72c/tests/TestCase/I18n/MessagesFileLoaderTest.php#L36-L49

I found a few other similar places. Shall I remove it there as well? Would that require a different pull request?

Reply:
> Shall I remove it there as well? Would that require a different pull request?

You can update the other backup/restore pattern usage if you want. It doesn't need to be a separate pull request.

Reply:
Done!